In [1]:
import pandas as pd

In [2]:
import numpy as np

In [3]:
data = pd.DataFrame()

In [4]:
data['age'] = 14, 12, 11, 10, 8, 6, 8

In [6]:
data

age
0   14
1   12
2   11
3   10
4    8
5    6
6    8

In [8]:
data.mean()

age    9.857143
dtype: float64

In [9]:
data.age.mean()

9.857142857142858

In [10]:
data.median()

age    10.0
dtype: float64

In [11]:
data.mode()

age
0    8

In [12]:
data.age.mode()

0    8
dtype: int32

In [15]:
data.var()

age    7.47619
dtype: float64

In [40]:
np.std(data, ddof=1)

age         2.734262
5           0.000000
5           0.000000
(5, age)    0.000000
dtype: float64

In [41]:
data.std()

age         2.734262
5           0.000000
5           0.000000
(5, age)    0.000000
dtype: float64

In [20]:
np.std(data ,ddof=1) / np.sqrt(len(data['age']))

age    1.033454
dtype: float64

In [21]:
data

age
0   14
1   12
2   11
3   10
4    8
5    6
6    8

In [31]:
data2 = pd.DataFrame()

In [32]:
data2['age'] = 14, 12, 11, 10, 8, 7, 8

In [33]:
data2.var()

age    6.333333
dtype: float64

In [34]:
data2.mean()

age    10.0
dtype: float64

In [35]:
data2.mode()

age
0    8

In [36]:
data2.median()

age    10.0
dtype: float64

In [37]:
data2.std()

age    2.516611
dtype: float64

In [42]:
np.std(data2['age'] ,ddof=1) / np.sqrt(len(data2['age']))

0.95118973121134176

In [44]:
data3 = pd.DataFrame()

In [45]:
data3['age'] = 14, 12, 11, 10, 8, 7, 1

### This does not change my choices.

In [46]:
data4 = pd.DataFrame()

In [47]:
data4['percent_like'] = .2, .23, .17, .05

In [48]:
data4.mean()

percent_like    0.1625
dtype: float64

### I would estimate 16.25% of the total population are Brady Bunch fans on the 50th Anniversary.